package.json

```diff
@@ -21,10 +24,12 @@
     "@babel/plugin-transform-runtime": "7.18.2",
     "@babel/preset-react": "7.17.12",
     "html-webpack-plugin": "5.5.0",
+    "husky": "8.0.3",
```

Comment:
do we need lint staged?

Reply:
Yes we do, speeds up the precommit hook - it's not such a big difference when only running prettier but once you add linting in there as well it's quite noticeable.  Also ideally we have the same basic tooling across all our repos.